src/components/Error/style.css

```diff
@@ -0,0 +1,32 @@
+.error {
```

Comment:
Minor: you can make your selectors simpler if you like, considering that all will already be namespaced with `Error`.

Something like  `.container, .message, .instruction, .icon`